ej2-angular/daterangepicker/getting-started.md

```diff
@@ -1,14 +1,14 @@
 ---
 layout: post
-title: Getting started with Angular Daterangepicker component | Syncfusion
-description:  Checkout and learn about Getting started with Angular Daterangepicker component of Syncfusion Essential JS 2 and more details.
+title: Getting Started with Angular Date Range Picker Component | Syncfusion
+description: Learn here all about getting started with Syncfusion Essential Angular Date Range Picker component, it's elements and more.
```

Comment:
Revert this and remove unwanted space only.

Reply:
u[dated